packages/components/src/components/PipelineRuns/PipelineRuns.js

```diff
@@ -97,51 +97,59 @@ const PipelineRuns = ({
     }
   });
 
-  const pipelineRunsFormatted = pipelineRuns.map(pipelineRun => ({
-    id: `${pipelineRun.metadata.namespace}:${pipelineRun.metadata.name}`,
-    name: (
-      <Link
-        to={createPipelineRunURL({
-          namespace: pipelineRun.metadata.namespace,
-          pipelineRunName: createPipelineRunDisplayName({
-            pipelineRunMetadata: pipelineRun.metadata
-          })
-        })}
-      >
-        {pipelineRun.metadata.name}
-      </Link>
-    ),
-    pipeline: !hidePipeline && (
-      <Link
-        to={createPipelineRunsByPipelineURL({
-          namespace: pipelineRun.metadata.namespace,
-          pipelineName: pipelineRun.spec.pipelineRef.name
-        })}
-      >
-        {pipelineRun.spec.pipelineRef.name}
-      </Link>
-    ),
-    namespace: !hideNamespace && pipelineRun.metadata.namespace,
-    status: (
-      <div className="definition">
-        <div
-          className="status"
-          data-status={getStatus(pipelineRun).status}
-          data-reason={getStatus(pipelineRun).reason}
-        >
-          <div className="status-icon">
-            {getPipelineRunStatusIcon(pipelineRun)}
+  const pipelineRunsFormatted = pipelineRuns.map(pipelineRun => {
+    const { namespace, annotations } = pipelineRun.metadata;
+    const pipelineRunName = createPipelineRunDisplayName({
+      pipelineRunMetadata: pipelineRun.metadata
+    });
+    const pipelineRefName = pipelineRun.spec.pipelineRef.name;
```

Comment:
do we still have an issue here?

Reply:
noup this line takes care of undefined values: https://github.com/tektoncd/dashboard/blob/0c73c4086cbf9f54ede001c4fa122007a0caac1e/packages/components/src/components/PipelineRuns/PipelineRuns.js#L120

Reply:
If `pipelineRun.spec.pipelineRef` is undefined wouldn't `pipelineRun.spec.pipelineRef.name` throw an error?

Reply:
double checked this and its just initializing `pipelineRefName` as undefined. no errors.

Reply:
strange that `pipelineRef` is present but as long as it's not causing any issues 🤷‍♂ let's keep an eye on it just in case